gitlab/v4/objects/iterations.py

```diff
@@ -16,11 +16,35 @@ class GroupIterationManager(ListMixin, RESTManager):
     _path = "/groups/{group_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"group_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # When using the API, the "in" keyword collides with python's "in" keyword
+    # raising a SyntaxError.
+    # For this reason, we have to use the query_parameters argument:
+    # group.iterations.list(query_parameters={"in": "title"})
+    _list_filters = (
+        "include_ancestors",
+        "include_descendants",
+        "in",
+        "search",
+        "state",
+        "updated_after",
+        "updated_before",
+    )
 
 
 class ProjectIterationManager(ListMixin, RESTManager):
     _path = "/projects/{project_id}/iterations"
     _obj_cls = GroupIteration
     _from_parent_attrs = {"project_id": "id"}
-    _list_filters = ("state", "search", "include_ancestors")
+    # When using the API, the "in" keyword collides with python's "in" keyword
+    # raising a SyntaxError.
+    # For this reason, we have to use the query_parameters argument:
+    # group.iterations.list(query_parameters={"in": "title"})
+    _list_filters = (
+        "include_ancestors",
+        "include_descendants",
+        "in",
+        "search",
+        "state",
+        "updated_after",
+        "updated_before",
+    )
```

Comment:
This will also require `from gitlab import types` at the top:

```suggestion
    )
    _types = {"in": types.ArrayAttribute}
```